src/pages/choose-renewable-energy/index.js

```diff
@@ -65,16 +65,17 @@ const ChooseRenewablePage = () => (
               href="#actions"
               title="See how you can take action"
             >
-              <button
+              <a
                 className={`${ButtonStyles.btnSimple} ${ButtonStyles.btnMd} ${
                   ButtonStyles.btnBanner
                 } ${ButtonStyles.btnMobile}`}
+                aria-label={"Learn what you can do"}
               >
                 Learn what you can do
-              </button>
+              </a>
             </a>
             <div className={ChoiceStyles.bannerCtaSecondary}>
-              <a href="#how" title="How it happens">
+              <a href="/choose-renewable-energy/#how" title="How it happens">
```

Comment:
This works however it reload the page rather than just scrolling down to the section. 

I believe the href needs to read `href="#how"` and it's associated id needs changed to be `id="how"`

Reply:
This appears to be how it works currently at https://climatechoice.co/change-how-you-eat, unless I'm misunderstanding. When you click on one of the 'how do I x' link it brings you back to the top. 

So I think we should raise a separate task to figure that out, as this task was for the accessibility issues. Thanks for all your comments by the way, this is my first real attempt at an open-source project and it's been a great experience.